app/feature/feature-edit-coinsured/src/main/kotlin/com/hedvig/android/feature/editcoinsured/data/CoInsured.kt

```diff
@@ -1,11 +1,22 @@
 package com.hedvig.android.feature.editcoinsured.data
 
+import java.time.format.DateTimeFormatter
 import kotlinx.datetime.LocalDate
+import kotlinx.datetime.toJavaLocalDate
 
 data class CoInsured(
   val firstName: String?,
   val lastName: String?,
   val birthDate: LocalDate?,
   val ssn: String?,
   val hasMissingInfo: Boolean,
-)
+) {
+  val displayName: String? = if (firstName != null && lastName != null) {
+    "$firstName $lastName"
+  } else {
+    null
+  }
```

Comment:
Potentially same comment as two PRs before, with the name and/or last name showing individually if either of them are non-null